Feature/new iasi omps

SteveStegall-NOAA opened the pull request:
Hello,

I have updated the fix files for OMPS and IASI for your review.

Thanks.

Steve Stegall

SteveStegall-NOAA committed:
Updated for new HEAP IASI data

SteveStegall-NOAA committed:
Updated

SteveStegall-NOAA committed:
Updated for new OMPS data

SteveStegall-NOAA committed:
Updated for release in July

ilianagenkova commented on fix/airs:
```diff
@@ -19,5 +19,6 @@ export HOURS2ALARM=${HOURS2ALARM:-6}
 #PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"AIRS_efov IASI_efov MT_saphir AMSR2"}
-PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov AMSR2"}
+#PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov AMSR2"}
+PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov"}
```
Why is AMSR2 family removed?

ilianagenkova reviewed (commented):
@SteveStegall-NOAA, could you walk me through what's in the /TRIGGERS directory? I assume its content is only for us, ObsProc team, but I would like to understand the need for two new JISATINGEST_trig_* triggers and two lsf.JISATINGEST_trig_* files. thanks, iliana

SteveStegall-NOAA commented:
Hello,

I did not realize I committed the fix/airs file.  I only changed it to
ingest the IASI-efov data only for testing and to limit PDA pulls.  I will
change the fix file back to ingest all the families.

Thanks.

Steve Stegall



On Mon, Jun 27, 2022 at 6:09 PM iliana Genkova ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> @SteveStegall-NOAA <https://github.com/SteveStegall-NOAA>, could you walk
> me through what's in the /TRIGGERS directory? I assume its content is only
> for us, ObsProc team, but I would like to understand the need for two new
> JISATINGEST_trig_* triggers and two lsf.JISATINGEST_trig_* files. thanks,
> iliana
> ------------------------------
>
> In fix/airs
> <https://github.com/NOAA-EMC/satingest/pull/12#discussion_r907835693>:
>
> > @@ -19,5 +19,6 @@ export HOURS2ALARM=${HOURS2ALARM:-6}
>  export log=${log:-$OUTDIR/tranjb_ingest_airs.out}
>
>  #PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"AIRS_efov IASI_efov MT_saphir AMSR2"}
> -PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov AMSR2"}
> +#PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov AMSR2"}
> +PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov"}
>
> Why is AMSR2 family removed?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/satingest/pull/12#pullrequestreview-1020835426>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWBGV2UB7R5QEMT3CQ645ETVRIRBVANCNFSM5ZX6WHNA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

I.M. Systems Group
IMSG at NOAA/NWS/NCEP/EMC
5830 University Research Ct., Rm. 2016
College Park, MD 20740

301-683-3956

***@***.***

SteveStegall-NOAA committed:
Update airs

SteveStegall-NOAA commented:
Hello,  I changed the fix/airs file and committed it.  You can review it
when you have a chance.  Thanks.

Steve Stegall

On Tue, Jun 28, 2022 at 9:20 AM Steve Stegall - NOAA Affiliate <
***@***.***> wrote:

> Hello,
>
> I did not realize I committed the fix/airs file.  I only changed it to
> ingest the IASI-efov data only for testing and to limit PDA pulls.  I will
> change the fix file back to ingest all the families.
>
> Thanks.
>
> Steve Stegall
>
>
>
> On Mon, Jun 27, 2022 at 6:09 PM iliana Genkova ***@***.***>
> wrote:
>
>> ***@***.**** commented on this pull request.
>>
>> @SteveStegall-NOAA <https://github.com/SteveStegall-NOAA>, could you
>> walk me through what's in the /TRIGGERS directory? I assume its content is
>> only for us, ObsProc team, but I would like to understand the need for two
>> new JISATINGEST_trig_* triggers and two lsf.JISATINGEST_trig_* files.
>> thanks, iliana
>> ------------------------------
>>
>> In fix/airs
>> <https://github.com/NOAA-EMC/satingest/pull/12#discussion_r907835693>:
>>
>> > @@ -19,5 +19,6 @@ export HOURS2ALARM=${HOURS2ALARM:-6}
>>  export log=${log:-$OUTDIR/tranjb_ingest_airs.out}
>>
>>  #PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"AIRS_efov IASI_efov MT_saphir AMSR2"}
>> -PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov AMSR2"}
>> +#PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov AMSR2"}
>> +PROCESSING_FAMILIES=${PROCESSING_FAMILIES:-"IASI_efov"}
>>
>> Why is AMSR2 family removed?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/satingest/pull/12#pullrequestreview-1020835426>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AWBGV2UB7R5QEMT3CQ645ETVRIRBVANCNFSM5ZX6WHNA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> I.M. Systems Group
> IMSG at NOAA/NWS/NCEP/EMC
> 5830 University Research Ct., Rm. 2016
> College Park, MD 20740
>
> 301-683-3956
>
> ***@***.***
>
>

-- 

I.M. Systems Group
IMSG at NOAA/NWS/NCEP/EMC
5830 University Research Ct., Rm. 2016
College Park, MD 20740

301-683-3956

***@***.***

ShelleyMelchior-NOAA commented:
This PR seems fine to me. But I don't advise adding the specifically tailored trigger scripts. Keep those local.

ilianagenkova commented:
Won't we need to update the one trigger we gave NCO at hand over (Deec 2021), i.e. merge the changes from JISATINGEST_trig_IASI into the JISATINGEST ?

SteveStegall-NOAA committed:
Update ingest_sncvgrib.sh

SteveStegall-NOAA committed:
Update ingest_sncvgrib.sh

These lines are updated:
241:  #rm -f $raw_file  # remove IMS snow cover/sea ice file from temporary working directory
242:  cp $raw_file $TANKDIR/$yyyymmdd/wgrbbul/  #copy IMS snow cover/sea ice file to dir for user to pick up

ShelleyMelchior-NOAA commented:
> Won't we need to update the one trigger we gave NCO at hand over (Deec 2021), i.e. merge the changes from JISATINGEST_trig_IASI into the JISATINGEST ?

I think the JISATINGEST_trig_[IASI|OMPS] only consists of local changes Steve made for this tasking (e.g., path changes). The original JISATINGEST trigger is general . But it looks like the general trigger script in the repository has "Sudhir.Nadiga" hard wired in it. I don't mind merging $USER that Steve has in his triggers. That is a good plan.

ilianagenkova reviewed (approved):
Changes look good!
